src/Microsoft.Data/Microsoft.Data.DataFrame.csproj

```diff
@@ -153,7 +153,7 @@
 
   <ItemGroup>
     <EmbeddedResource Update="Strings.resx">
-      <Generator>PublicResXFileCodeGenerator</Generator>
+      <Generator>ResXFileCodeGenerator</Generator>
```

Comment:
What's the story here?

Reply:
Changes the `Strings.resx` modifier to private

Reply:
It didn't do that in this PR....

Reply:
Reverted this